ddprof-ffi/src/exporter.rs

```diff
@@ -73,7 +74,7 @@ pub extern "C" fn endpoint_agent(base_url: CharSlice) -> EndpointV3 {
 /// # Arguments
 /// * `site` - Contains a host and port e.g. "datadoghq.com".
 /// * `api_key` - Contains the Datadog API key.
-#[export_name = "ddprof_ffi_EndpointV3_agentless"]
+#[export_name = "ddog_EndpointV3_agentless"]
```

Comment:
Arguably this could be used for agentless as well in telemetry (I think they're planning on supporting it?); the two things that go here (site and api key) are what they'd need as well.